setup.py

```diff
@@ -49,6 +49,8 @@ def read(*rnames):
           'Programming Language :: Python :: 3.4',
```

Comment:
Consider dropping support for EOL Python 3.4.

Reply:
Good catch!

Reply:
It was not tested in `.travis.yml` and `tox.ini`, so was blindly following the content. Thanks for the Review.

Reply:
Shameless plug: https://pypi.org/project/check-python-versions/ can catch inconsistencies like that (and automate Python version updates in all the files).

Reply:
That looks very interesting and might end up in my package maintainer venv.